docs/installation/production/docker/tethys_docker_reference.rst

```diff
@@ -127,7 +127,12 @@ These environment variables point to important paths in the container. Changing
 |                           | :file:`portal_config.yml`. Defaults to "${TETHYS_PERSIST}/static"                        |
 +---------------------------+------------------------------------------------------------------------------------------+
 | WORKSPACE_ROOT            | Path to the tethys workspaces root folder. This also sets the associated setting in the  |
-|                           | :file:`portal_config.yml`. Defaults to "${TETHYS_PERSIST}/workspaces"                    |
+|                           | :file:`portal_config.yml`. Defaults to "/media/"                                         |
```

Comment:
I'm looking at this on my phone, so it's hard to tell, but I think you modified the wrong line here. This should be on MEDIA_ROOT.

Reply:
fixed